lib/puppet/provider/package/aix.rb

```diff
@@ -106,8 +119,12 @@ def self.pkglist(hash = {})
     end
 
     begin
-      list = execute(cmd).scan(/^[^#][^:]*:([^:]*):([^:]*)/).collect { |n,e|
-        { :name => n, :ensure => e, :provider => self.name }
+      list = execute(cmd).scan(/^[^#][^:]*:([^:]*):([^:]*):[^:]*:[^:]*:([^:])/).collect { |n,e,s|
```

Comment:
I see this was used before, but can we use more meaningful variables than `n, e, s`? like name, ensure, status.

Reply:
I don't think we can use ensure since it's a reserved keyword in ruby.